packages/sampler/stories/default/VirtualList.js

```diff
@@ -54,8 +46,8 @@ storiesOf('UI', module)
 				<VirtualList
 					dataSize={updateDataSize(number('dataSize', VirtualListConfig, defaultDataSize))}
 					horizontalScrollbar={select('horizontalScrollbar', prop.scrollbarOption, VirtualListConfig)}
-					itemRenderer={uiRenderItem(ri.scale(number('itemSize', VirtualListConfig, 72)))}
-					itemSize={ri.scale(number('itemSize', VirtualListConfig, 72))}
+					itemRenderer={uiRenderItem(ri.scale(number('itemSize', VirtualListConfig, 156)))}
+					itemSize={ri.scale(number('itemSize', VirtualListConfig, 156))}
```

Comment:
![image](https://user-images.githubusercontent.com/34676178/76291355-43f4a100-62f0-11ea-962d-1b53dddfd400.png)

I don't think we should update it. 
Let's update sandstone only.

Reply:
I agree.

Reply:
Let's not change 72 to 156 here.